src/server/editorServices.ts

```diff
@@ -541,174 +576,162 @@ namespace ts.server {
         }
 
         private updateProjectGraphs(projects: Project[]) {
-            let shouldRefreshInferredProjects = false;
             for (const p of projects) {
                 if (!p.updateGraph()) {
-                    shouldRefreshInferredProjects = true;
+                    this.pendingInferredProjectUpdate = true;
                 }
             }
-            if (shouldRefreshInferredProjects) {
-                this.refreshInferredProjects();
-            }
         }
 
-        private onSourceFileChanged(fileName: NormalizedPath) {
+        private onSourceFileChanged(fileName: NormalizedPath, eventKind: FileWatcherEventKind) {
             const info = this.getScriptInfoForNormalizedPath(fileName);
             if (!info) {
                 this.logger.info(`Error: got watch notification for unknown file: ${fileName}`);

```

Comment:
I think its new and didnt exist before, i will fix this when merging master in this branch